src/paginator.ts

```diff
@@ -37,6 +37,7 @@ export class Paginator<Params, Result>
       typeof response.headers?.link === 'string'
         ? this.pluckNext(response.headers.link)
         : undefined;
+    this.nextParams = {};
```

Comment:
Typecheck seems to be failing. You can just add downcasting such as `{} as Params` for now

Reply:
Fixed.